infrastructure/redis/src/main/kotlin/com/studentcenter/weave/infrastructure/redis/meetingTeam/entity/MeetingTeamInvitationRedisHash.kt

```diff
@@ -0,0 +1,27 @@
+package com.studentcenter.weave.infrastructure.redis.meetingTeam.entity
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.redis.core.RedisHash
+import org.springframework.data.redis.core.TimeToLive
+import java.util.*
+import kotlin.time.Duration
+
+@RedisHash("meeting_team_invitation")
+class MeetingTeamInvitationRedisHash(
+    teamId: UUID,
+    invitationLink: String,
+    expirationDuration: Duration,
+) {
+
+    @Id
+    var invitationLink: String = invitationLink
+        private set
+
+    var teamId: UUID = teamId
+        private set
+
+    @TimeToLive
+    var expirationDuration: Duration = expirationDuration
+        private set
```

Comment:
p3) Duration 정상적으로 동작하나요?
```suggestion
    @TimeToLive
    var expirationDuration: Duration = expirationDuration
        private set
```